app/resources/start.html.template

```diff
@@ -18,18 +18,18 @@
   <div class="main">
     <div class="row">
       <div class="action-outer start">
-        <div class="action-inner">
-          <a href="{{seek_url}}" tabindex="1">
+        <a href="{{seek_url}}" tabindex="1">
```

Comment:
This is actually invalid in HTML standard. A tag is an inline element while DIV tag is a block element, and an inline element cannot contain a block element. See e.g.:
http://www.htmq.com/htmlkihon/005.shtml
It actually works in most browsers, but we want to be HTML standard compliant when possible.

It seems nontrivial to make a whole DIV tag as a link. It is discussed here:
https://stackoverflow.com/questions/796087/make-a-div-into-a-link

You can read and try answers there to see which works best for us.

https://stackoverflow.com/a/3494108 is popular in that entry, but I personally don't like it so much :) It looks too hacky and is not very readable.

https://stackoverflow.com/a/796096 looks like a simple good solution. But not sure if we can apply it together with our action-outer/action-inner technique, which is used to centerize the text.

https://stackoverflow.com/a/796094 is an answer many people dislike, but I feel it's not so bad in our case. You may fall back to this solution.